install/infra/modules/gke/output.tf

```diff
@@ -38,15 +38,20 @@ output "registry" {
     url      = data.google_container_registry_repository.gitpod[0].repository_url
     server   = regex("[^/?#]*", data.google_container_registry_repository.gitpod[0].repository_url)
     username = "_json_key"
-    password = "Copy paste the content of the service account credentials JSON file"
+    password = base64decode(google_service_account_key.obj_sa_key[0].private_key)
   }, "No container registry created")
 }
 
+output "dns_credentials" {
+  sensitive = false
```

Comment:
**question:** The other outputs are marked as sensitive, why is this one marked as not sensitive?

Reply:
Good point! making this sensitive as well. Previously this was just the name of the file, at which point it was not very sensitive as an info.